src/asciidoc/docs/_includes/steps-codeeditor.adoc

```diff
@@ -0,0 +1,23 @@
+= Steps for the Code Editor
+
+////
+  Usage:
+    :includespath: ../_includes
+
+    include::{includespath}/steps-codeeditor.adoc[tag=openproject-template2edit]
+    include::{includespath}/steps-codeeditor.adoc[tag=expand-terminal]
+////
+
+
+// tag::all[]
+
+// tag::openproject-template2edit[]
+. In https://withkoji.com/developer/projects[your developer portal page], open the project associated with the template to edit.
+// end::openproject-template2edit[]
+
+// tag::expand-terminal[]
+. Expand the terminal pane of the Code Editor.
```

Comment:
I wouldn't use an include for this step. It seems like it wouldn't be used that frequently, is easy to replicate, and creates more maintenance overhead than value. Also, I would shorten it, since the previous step should be to open the code editor.
```suggestion
. Expand the terminal pane.
```

Reply:
Fixed. Will be in the next commit.